translations/introduction.tex/ua.po

```diff
@@ -72,6 +79,29 @@ msgid ""
 "}\n"
 "\\vfill"
 msgstr ""
+"{2}\n"
+"\\textbf{Примітка}: У цій книзі правил, ігрові терміни та компоненти пишуться з Великої літери.\n"
+"\\textbf{Жирний текст} використовується,щоб привернути увагу до важливих правил\n"
+"\\textit{IКурсивом} виділено приклади ігрового процесу.\n"
+"\\hyperlink{Heroes of Might and Magic III}{Гіперпосилання коричневого кольору} ведуть до інших частин книги правил.\n"
+"% The following glyphs MUST be rendered before their first use in tabular environment in all_map_locations.tex.\n"
+"% Inkscape will fail otherwise.\n"
+"% Other glyphs referenced over there ONLY cause no issues ¯\\_(ツ)_/¯\n"
+"\\phantom{\n"
+"  \\includesvg[height=0.1px]{\\svgs/2_treasure_die.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/arrow_right_gray.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/artifact.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/damage-table.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/movement.svg}\n"
+"  \\includesvg[height=0.1px]{\\svgs/spellpower.svg}\n"
+"}\n"
+"\\vfill\n"
+"\\columnbreak\n"
+"\\note{11}{Винятки та примітки з \\hyperlink{Heroes of Might and Magic III}{гіперпосиланнями} пояснюються у вікнах подібних до цього.\n"
```

Comment:
```suggestion
"\\note{10}{\\vspace{1em}Винятки та примітки з \\hyperlink{Heroes of Might and Magic III}{гіперпосиланнями} пояснюються у вікнах подібних до цього.\n"
```

Reply:
This is how it looks like with the change:
![ua-04](https://github.com/Heegu-sama/Homm3BG/assets/5611195/3fc2cdbf-b137-4f67-b594-e1c7e7a81778)

Reply:
This is how it looks like with the change:

![ua-04](https://github.com/Heegu-sama/Homm3BG/assets/5611195/c85a87c7-c02c-4fb5-bbbb-5a7ce0ebb532)